internal/backend/runtime/omni/pkg/check/etcd.go

```diff
@@ -93,12 +93,28 @@ func Etcd(ctx context.Context, r controller.Reader, clusterName string) error {
 		}
 	}
 
-	if (len(members)-1)%2 != 0 {
+	quorum := len(members)/2 + 1
+
+	switch {
+	case len(members) == quorum && len(members) != 1:
+		return newError(
+			specs.ControlPlaneStatusSpec_Condition_Warning,
+			false,
+			"Etcd members count is equal to the minimum quorum %d. The cluster cannot tolerate the loss of any members",
```

Comment:
I don't think this condition is reachable except for N = 2.

Reply:
yeah, can simplify